portfolio/src/main/webapp/index.html

```diff
@@ -26,11 +29,11 @@ <h5>Welcome to my portfolio :)</h5>
         &nbsp;
 
         <!-- header -->
-        <div class = "header-pic">
+        <!-- <div class = "header-pic">
```

Comment:
prefer not to leave code commented out. Either delete it or link to an issue in a TODO comment

Reply:
Okay!